tests/integrations/gcp/test_gcp.py

```diff
@@ -5,21 +5,22 @@
 import json
 import time
 from textwrap import dedent
-import requests
 import uuid
 import tempfile
 import shutil
 import sys
 import subprocess
 
 import pytest
-import pickle
 import os.path
 import os
 from googleapiclient.discovery import build
 from google_auth_oauthlib.flow import InstalledAppFlow
 from google.auth.transport.requests import Request
 
+requests = pytest.importorskip("requests")
+pickle = pytest.importorskip("pickle")
```

Comment:
pickle is stdlib, this should not be necessary

Reply:
okay, I'll change this.